atmosphere/ffc0c6e274094c6423e29d1c45bab1593b8fac7d/README.md

```diff
@@ -4,43 +4,73 @@
 ## 2. Checkout to merge the commit hash
     git checkout ffc0c6e274094c6423e29d1c45bab1593b8fac7d
 
-## 3. Edit the version tag the _pom.xml_ at _all the project_, searching for pom.xml at root project and remove _-SNAPSHOT_ version
+## 3. Edit the build tag (```<build> <plugins> ```  HERE ... ```</plugins> </build>```) on the _pom.xml_ at the directory **atmosphere/modules/cpr**:
```

Comment:
In this case, you replaced the contents of the plugin tag, right? 
If so, you might inform it. For example: Edit the build tag by replacing the old plugin tag with the new one presented below.

Reply:
No, in this case I just add the plugin that is mentioned below in the pom file.